Makefile

```diff
@@ -360,6 +396,12 @@ $(KUSTOMIZE): # Build kustomize from tools folder.
 $(SETUP_ENVTEST): # Build setup-envtest from tools folder.
 	GOBIN=$(TOOLS_BIN_DIR) $(GO_INSTALL) $(SETUP_ENVTEST_PKG) $(SETUP_ENVTEST_BIN) $(SETUP_ENVTEST_VER)
 
+$(GINKGO): # Build ginkgo from tools folder.
+	GOBIN=$(TOOLS_BIN_DIR) $(GO_INSTALL) $(GINKGO_PKG) $(GINKGO_BIN) $(GINGKO_VER)
+
+$(CAPI_OPERATOR):
+	wget $(CAPI_OPERATOR_URL) -O $(CAPI_OPERATOR)
```

Comment:
do we download chat because it's installed from code in e2e setup?

Reply:
Yes, I’d like to add a way to override CAPI operator image with a custom chart, if there will be some major issues during testing we didn’t saw yet, but could be hot-fixed temporarily

Reply:
I think this one could be closed.